en-EN.js

```diff
@@ -142,31 +142,31 @@ module.exports = {
             fetch_error: "**Whoops!** The required data could not be retrieved to create a graph.. *If the problem happens again, please report the bug on the support server.*"
         },
         help_screen: {
-            embed_title: "Créer un graphique",
-            embed_description: "**Représentez** vos statistiques ou celles du serveur sous forme de graphiques. Commencez par choisir une **préconfiguration** :",
-            image_title: "Graphiques",
-            embed_field_server_graph: "Graphiques du serveur",
-            embed_field_user_graph: "Graphique de vos stats",
-            components_guild_graph_select_menu: "Choisir un graphique sur les stats du serveur..",
-            components_user_graph_select_menu: "..ou un graphique avec vos statistiques !"
+            embed_title: "Create a graph",
+            embed_description: "**Display** your stats or the server's stats with graphs. Start by selecting a **configuration** :",
+            image_title: "Graphs",
+            embed_field_server_graph: "Server graphs",
+            embed_field_user_graph: "Your stats' graphs",
+            components_guild_graph_select_menu: "Select a server stats graph..",
+            components_user_graph_select_menu: "..or a graph with your stats!"
         },
         graph_main_screen: {
             advanced: {
-                confirm_button: "Générer",
-                advanced_button: "Options avancées",
-                advanced_embed_description: "**Adaptez** et **personnalisez** l'interprétation et l'affichage de vos graphiques.",
-                advanced_embed_options_field_title: "Options disponibles :",
+                confirm_button: "Generate",
+                advanced_button: "Advanced settings",
+                advanced_embed_description: "**Adapt** and **customize** your graphs' interpretation and display layout.",
```

Comment:
`**Adapt** and **customize** the display layout and dataset representation.` Me parait plus adapté